packages/qwik/src/core/shared/scheduler.ts

```diff
@@ -511,11 +492,8 @@ function choreComparator(
           You are attempting to change a state that has already been streamed to the client.
           This can lead to inconsistencies between Server-Side Rendering (SSR) and Client-Side Rendering (CSR).
           Problematic Node: ${aHost.toString()}`;
-        if (shouldThrowOnHostMismatch) {
-          throw qError(QError.serverHostMismatch, [errorMessage]);
-        }
         logWarn(errorMessage);
-        return null;
+        return 1;
```

Comment:
this makes sorting unstable. If you don't see a difference, return `0`.

Reply:
actually shouldn't we still sort by host depth?

and the warning should happen when ssrnodes that already streamed are marked dirty, not when tasks rerun, no?

Reply:
for some reason returning 0 is not working, I will look at it